api_spec/edge_api.json

```diff
@@ -95,23 +95,23 @@
         "/bulk-identities": {
             "post": {
                 "tags": [],
-                "summary": "Post bulk identities",
-                "description": "Post traits for a given user.",
+                "summary": "Bulk-Identities",
```

Comment:
this was probably better before, we should keep it consistent with the rest of the summaries: 

<img width="306" alt="image" src="https://user-images.githubusercontent.com/14089968/200912348-db5dd118-35d0-4dcc-a987-b0f36937f43a.png">

Perhaps something like: "Bulk insert identities" would be good.